ee/clickhouse/views/person.py

```diff
@@ -32,6 +33,10 @@
 from posthog.utils import format_query_params_absolute_url
 
 
+def should_paginate(results, limit: Union[str, int]) -> bool:
+    return len(results) > cast(int, limit) - 1
```

Comment:
instead of just typecasting, shouldn't this try to parse limit if its a string? O.o